setup.cfg

```diff
@@ -19,6 +19,8 @@ exclude=.tox,venv,awx/lib/site-packages,awx/plugins/inventory/ec2.py,awx/plugins
 max-line-length=160
 ignore=E201,E203,E221,E225,E231,E241,E251,E261,E265,E303,W291,W391,W293,E731,F405
 exclude=.tox,venv,awx/lib/site-packages,awx/plugins/inventory,awx/ui,awx/api/urls.py,awx/main/migrations,awx/main/south_migrations,awx/main/tests/data,node_modules/,awx/projects/,tools/docker,awx/settings/local_*.py,installer/openshift/settings.py,build/,installer/
+per-file-ignores =
```

Comment:
TIL you can do this.

Reply:
One can also ignore errors inline, with a line like:

```python
print('foo')  # noqa: E731,E123
```

That'd be more maintainable over time, as lines of code shift around. Less spooky action at a distance.

Reply:
There was like twenty long lines in that file and I didn't really want to ignore them one by one